src/main/java/com/ylysenkova/movieland/model/Movie.java

```diff
@@ -10,10 +11,13 @@
     private int yearOfRelease;
     private String description;
     private double rating;
-    private double price;
+    private BigDecimal price;
```

Comment:
double was better, you need BigDecimal for calculation only. But calculation may not be requested. So weight of BigDecimal is bigger.